db/seeds/development.rb

```diff
@@ -0,0 +1,2 @@
+byebug
```

Comment:
watch out. 

also, I wouldn't add any user in the dev seeds, only the first admin user (that we dont have yet) but not a harcoded user